pallets/parachain-system/src/consensus_hook.rs

```diff
@@ -100,3 +102,15 @@ impl<const N: u32> ConsensusHook for FixedCapacityUnincludedSegment<N> {
 ///
 /// This is a simple type alias around a fixed-capacity unincluded segment with a size of 1.
 pub type RequireParentIncluded = FixedCapacityUnincludedSegment<1>;
+
+impl<T: pallet_aura_ext::Config, const V: u32, const C: u32> ConsensusHook
+	for aura_consensus_hook::FixedVelocityConsensusHook<T, V, C>
+{
+	fn on_state_proof(_state_proof: &RelayChainStateProof) -> UnincludedSegmentCapacity {
+		Self::validate_slot();
```

Comment:
This uses weight so we should probably modify consensus hook to return it

Reply:
This implementation should be in the `pallet-aura-ext` crate. The dependency is backwards.